root/static/scripts/common/utility/arrays.js

```diff
@@ -111,7 +111,7 @@ export function sortByString<T>(
 export function groupBy<T>(
   array: $ReadOnlyArray<T>,
   func: (T) => string,
-): {__proto__: null, +[groupKey: string]: $ReadOnlyArray<T>} {
+): {__proto__: empty, +[groupKey: string]: $ReadOnlyArray<T>} {
```

Comment:
Why? (sorry if these `__proto__` changes should be obvious, I've just never dealt with that at all AFAICT)

Reply:
As explained above, it lets you use the result where `__proto__: null` would conflict with another object type that has a normal object prototype.